x-pack/plugins/ml/public/embeddables/job_creation/aiops/flyout/create_job.tsx

```diff
@@ -0,0 +1,268 @@
+      jobId,
+      bucketSpan,
+      dataView,
+      field,
+      partitionField,
+      stopOnWarn,
+      query,
+      timeRange,
+      startJob,
+      runInRealTime
+    );
+    return result;
+  }
+  return (
+    <JobDetails
+      createADJob={createADJob}
+      createADJobInWizard={createADJobInWizard}
+      embeddable={undefined}
+      timeRange={timeRange}
+      layer={undefined}
+      layerIndex={0}
+      outerFormComplete={formComplete}
+    >
+      <>
+        <EuiCheckableCard
+          id={'count'}
+          label={
+            <>
+              <EuiTitle size="xs">
+                <h5>
+                  <FormattedMessage
+                    defaultMessage="Count"
+                    id="xpack.ml.newJobFromPatternAnalysisFlyout.count.title"
+                  />
+                </h5>
+              </EuiTitle>
+              <EuiSpacer size="s" />
+              <FormattedMessage
+                defaultMessage="Look for anomalies in the event rate of a category."
+                id="xpack.ml.newJobFromPatternAnalysisFlyout.count.description"
+              />
+            </>
+          }
+          checked={categorizationType === CATEGORIZATION_TYPE.COUNT}
+          onChange={() => setCategorizationType(CATEGORIZATION_TYPE.COUNT)}
+        />
+
+        <EuiSpacer size="m" />
+
+        <EuiCheckableCard
+          id={'rare'}
+          label={
+            <>
+              <EuiTitle size="xs">
+                <h5>
+                  <FormattedMessage
+                    defaultMessage="Rare"
+                    id="xpack.ml.newJobFromPatternAnalysisFlyout.rare.title"
+                  />
+                </h5>
+              </EuiTitle>
+              <EuiSpacer size="s" />
+              <FormattedMessage
+                defaultMessage="Look for categories that occur rarely in time."
+                id="xpack.ml.newJobFromPatternAnalysisFlyout.rare.description"
+              />
+            </>
+          }
+          checked={categorizationType === CATEGORIZATION_TYPE.RARE}
+          onChange={() => setCategorizationType(CATEGORIZATION_TYPE.RARE)}
+        />
+        <EuiSpacer size="m" />
+        <EuiSwitch
+          name="categorizationPerPartitionSwitch"
+          disabled={false}
+          checked={enablePerPartitionCategorization}
+          onChange={toggleEnablePerPartitionCategorization}
+          data-test-subj="mlNewJobFromPatternAnalysisFlyoutSwitchCategorizationPerPartition"
+          label={
+            <FormattedMessage
+              id="xpack.ml.newJobFromPatternAnalysisFlyout.perPartitionCategorizationSwitchLabel"
+              defaultMessage="Enable per-partition categorization"
+            />
+          }
+        />
+
+        {enablePerPartitionCategorization ? (
+          <>
+            <EuiSpacer size="m" />
+
+            <EuiCallOut
+              size="s"
+              title="Determine categories independently for each value of the partition field."
```

Comment:
Shouldn't this be internationalized?

Reply:
Updated in https://github.com/elastic/kibana/pull/170567/commits/1afac7e44dfea0b35f940eaad0d719d7eac14972